db/seeds.rb

```diff
@@ -5,3 +5,13 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+wendy = User.create!(name: 'Wendy', photo: image2 , bio: lorem40, posts_counter: 0)
+john = User.create!(name: 'John', photo: image3, bio: lorem40, posts_counter: 0)
+
+post1 = wendy.posts.create!(title: 'Cats and Dogs', text: 'this is first post about cats and dogs!', comments_counter: 0, likes_counter: 0)
```

Comment:
- [ ] What I gave is a demo or hint that how you can add seed data to the database,  but there are some mistakes were there, Please fix them. I would recommend after adding the seed data always check by running `rails db:seed` if nothing is coming it means the seed is successfully run and no error is there, otherwise an error comes and you need to fix it. You can add more posts and comments if you like.
```ruby

# On line 9 and 10 - the `photo` value should be a string.

wendy = User.create!(name: 'Wendy', photo: 'image2' , bio: lorem40, posts_counter: 0)
john = User.create!(name: 'John', photo: 'image3', bio: lorem40, posts_counter: 0)

```